src/integration-test/java/org/openlmis/requisition/repository/RequisitionTemplateRepositoryIntegrationTest.java

```diff
@@ -391,4 +398,41 @@ private AvailableRequisitionColumn getColumnWithOption() {
   private AvailableRequisitionColumnOption getOption(String uuid) {
     return availableRequisitionColumnOptionRepository.findById(UUID.fromString(uuid)).orElse(null);
   }
+
+  @Test
+  public void shouldAllowDuplicationForRequisitionReportOnlyTemplateNames() {
+
+    RequisitionTemplate template = generateInstance();
+    template.requisitionReportingOnly();
+    repository.saveAndFlush(template);
+
+  }
+
```

Comment:
Other functions are separated only with single line, please delete this extra line

Reply:
Resolved